firstvoices/backend/permissions/predicates/base.py

```diff
@@ -135,3 +135,27 @@ def is_hidden_site(user, obj):
 has_member_access_to_site_obj = (
     has_at_least_member_membership & ~is_team_obj
 )  # noqa E1130
+
+
+#
+# Related dictionary_entry predicates
+# used in ImmersionLabels
+@predicate
+def has_related_public_dictionary_entry(user, obj):
+    return obj.dictionary_entry.visibility == Visibility.PUBLIC
+
+
+@predicate
+def has_related_team_dictionary_entry(user, obj):
+    return obj.dictionary_entry.visibility == Visibility.TEAM
+
+
+has_member_access_to_related_dictionary_entry = (
```

Comment:
If this predicate is meant to be used the same as `has_public_access_to_related_dictionary_entry` it is missing the `Predicate()` and also missing a name

Reply:
Yes, those have been removed now with the reuse of `is_visible_object` predicate.